keyboards/kbd8x/readme.md

```diff
@@ -0,0 +1,15 @@
+# KBD 8X
+
+![kbd8x](imgur.com image replace me!)
+
+TKL Keyboard from KBDFans
+
+Keyboard Maintainer: [MechMerlins](https://github.com/mechmerlin)  
+Hardware Supported: kbd8x 
```

Comment:
Needs an extra line feed/whatever here, as it's staying on one line.

Reply:
I apparently also spelled my name wrong.